generators/rgolang/index.js

```diff
@@ -3,9 +3,9 @@
 
 const generators = require("yeoman-generator");
 const fileReader = require("html-wiring");
+const pathUtil = require("path");
```

Comment:
Didn't just name this `path` because it is used in a context where the variable `path` is already used for a string.